src/main/java/stsjorbsmod/cards/cull/Discordance.java

```diff
@@ -0,0 +1,68 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Discordance extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Discordance.class);
+
+    private static final CardRarity RARITY = CardRarity.UNCOMMON;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = 1;
+
+
+
+    public Discordance() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        this.baseDamage = 0;
+        this.baseMagicNumber = magicNumber = 5;
```

Comment:
Minor suggestion: we've found it leads to fewer mistakes to use descriptively named constants for naming the "magicNumber" values; here, I'd probably add something like this under the `COST` definition:

```java
private static final int DAMAGE_PER_CARD_TYPE = 5;
private static final int UPGRADE_PLUS_DAMAGE_PER_CARD_TYPE = 2;
```

...and use those in place of the numbers in the ctor/upgrade methods.

Reply:
Updated per suggestion